src/application-base/ApplicationBase.jsx

```diff
@@ -107,50 +81,26 @@ const ApplicationBase = ({
     return () => {
       window.removeEventListener('beforeunload', onBeforeUnload);
     };
-  }, [unloadPromptIsDisabled, registeredPromptsRef]);
-
-  const { localeOverride } = useTestOverrides(); // Allows us to test deployed applications in different locales.
-
-  const theme = useMemo(() => ({
-    // If the theme class name is undefined or an empty string, that indicates we have the root theme and should apply the root theme name.
-    name: themeName || rootThemeName,
-    className: themeName,
-  }), [themeName]);
+  }, [unloadPromptIsDisabled]);
 
   return (
-    <div data-terra-application-base className={cx('application-base', { fill: !fitToParentIsDisabled })}>
-      <ThemeProvider
-        themeName={themeName}
-      >
-        <ThemeContextProvider theme={theme}>
-          <Base
-            customMessages={customTranslatedMessages}
-            translationsLoadingPlaceholder={translationsLoadingPlaceholder}
-            locale={localeOverride || locale || browserLocale}
-          >
-            <ApplicationErrorBoundary>
-              <ApplicationIntlProvider>
-                <ActiveBreakpointProvider>
-                  <NavigationPromptCheckpoint
-                    onPromptChange={(registeredPrompts) => {
-                      registeredPromptsRef.current = registeredPrompts;
-                    }}
-                  >
-                    <ApplicationLoadingOverlayProvider>
-                      <ApplicationStatusOverlayProvider>
-                        <Suspense fallback={<ApplicationLoadingOverlay isOpen />}>
-                          {children}
-                        </Suspense>
-                      </ApplicationStatusOverlayProvider>
-                    </ApplicationLoadingOverlayProvider>
-                  </NavigationPromptCheckpoint>
-                </ActiveBreakpointProvider>
-              </ApplicationIntlProvider>
-            </ApplicationErrorBoundary>
-          </Base>
-        </ThemeContextProvider>
-      </ThemeProvider>
-    </div>
+    <ThemeProvider themeName={themeName}>
+      <ThemeContextProvider theme={theme}>
+        <Base locale={localeOverride || locale || browserLocale}>
```

Comment:
This will also need this change: https://github.com/cerner/terra-application/pull/95